src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,11 +2,15 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger();
+    
     @Override
     public User login(String login, String password) throws AuthenticationException {
-        //TODO: add corresponding log message about method login was called
+        logger.info("Method login was called. Params: login={}", login);
```

Comment:
Not resolved. I asked to change log level to `debug` because you woudn't want to see this information in logs all the time, only when debugging for some bugs.